Lombiq.Analyzers.PowerShell/Rules/Measure-LineContinuation/Measure-LineContinuation.psm1

```diff
@@ -38,12 +38,12 @@ function Measure-LineContinuation
                     $PSItem.Kind -eq [System.Management.Automation.Language.TokenKind]::LineContinuation })
             {
                 $results += [Microsoft.Windows.Powershell.ScriptAnalyzer.Generic.DiagnosticRecord]@{
-                    "Extent" = $lineContinuationToken.Extent
-                    "Message" = 'Using backtick (line continuation) makes the code harder to read and' +
-                    ' maintain. Please consider using parameter splatting instead.'
-                    "RuleName" = "PSAvoidUsingLineContinuation"
-                    "RuleSuppressionID" = "PSAvoidUsingLineContinuation"
-                    "Severity" = "Warning"
+                    'Extent' = $lineContinuationToken.Extent
+                    'Message' = 'Using backtick (line continuation) makes the code harder to read and maintain.' +
+                    ' Please consider using parameter splatting instead.'
```

Comment:
Let's be bold:
```suggestion
                    ' Please use parameter splatting instead.'
```
If we generate a warning for this, we're not asking the user to consider - we're asking them to change the code.